python/pyspark/sql/tests/test_functions.py

```diff
@@ -55,6 +55,74 @@
 
 
 class FunctionsTests(ReusedSQLTestCase):
+    def test_function_parity(self):
+        # This test compares the available list of functions in pyspark.sql.functions with those
+        # available in the Scala/Java DataFrame API in org.apache.spark.sql.functions.
+        #
+        # NOTE FOR DEVELOPERS:
+        # If this test fails one of the following needs to happen
+        # * If a function was added to org.apache.spark.sql.functions it either needs to be added to
+        #     pyspark.sql.functions or added to the below expected_missing_in_py set.
+        # * If a function was added to pyspark.sql.functions that was already in
+        #     org.apache.spark.sql.functions then it needs to be removed from expected_missing_in_py
+        #     below. If the function has a different name it needs to be added to py_equiv_jvm
+        #     mapping.
+        # * If it's not related to an added/removed function then likely one of the exclusion lists
+        #     need updated.
+        from pyspark import SparkContext
+        from pyspark.sql import functions as py_functions
+        from inspect import getmembers, isfunction
```

Comment:
Let's just import it on the top

Reply:
I moved the imports to the top of the function. Not sure if you wanted that or global imports. I do see plenty of other tests that have some test specific imports at the beginning of their function.

Reply:
yeah, I know what you're doing is consistent with some places but I am trying to just move all imports as a global. I think we should just put all imports together as global ones.

Reply:
no problem, done